browser/src/slideshow/engine/SlideShowNavigator.ts

```diff
@@ -294,7 +294,11 @@ class SlideShowNavigator {
 	private clickHandler(aEvent: MouseEvent) {
 		if (aEvent.button === 0) {
 			const slideInfo = this.theMetaPres.getSlideInfoByIndex(this.currentSlide);
-			if (slideInfo.interactions.length == 0) {
+			if (
+				!slideInfo ||
+				!slideInfo.interactions ||
+				slideInfo.interactions.length == 0
+			) {
```

Comment:
LGTM but can't we use `slideInfo?.interactions?.length == 0`

Reply:
I think it is not enabled in the linter...